autoload/flog/state.vim

```diff
@@ -161,11 +162,15 @@ function! flog#state#GetCommitRefs(commit) abort
     " full: Full path including refs/.*/
     " path: Path with remote
     " tail: End of path only (no remote)
+    let l:remote = l:match[8][ : -2]
+    if index(l:remotes, l:remote) < 0
+      let l:remote = ""
```

Comment:
I believe that a another line should go before this to add the remote back to the tail since it's part of the branch name:

```
let l:tail = l:remote . "/" . l:tail
```